metrics/lib/features/common/presentation/metrics_theme/model/build_results_theme_data.dart

```diff
@@ -6,15 +6,19 @@ class BuildResultsThemeData {
   final Color successfulColor;
   final Color failedColor;
   final Color canceledColor;
+  final TextStyle titleStyle;
 
   /// Creates the new instance of the [BuildResultsThemeData].
   ///
   /// [successfulColor] is the color of the successful build bar.
   /// [failedColor] is the color of the failed build bar.
   /// [canceledColor] is the color of the canceled build bar.
+  /// [titleStyle] is the [TextStyle] of the bar graph title.
+  /// If nothing is specified the [ThemeData]'s default text color will be used.
```

Comment:
Probably we shouldn't clarify how other classes will work, this class doesn't contain described logic so probably this comment should be moved elsewhere.